docs/reference/data-frames/apis/put-transform.asciidoc

```diff
@@ -45,6 +45,17 @@ IMPORTANT:  You must use {kib} or this API to create a {dataframe-transform}.
   can contain lowercase alphanumeric characters (a-z and 0-9), hyphens, and
   underscores. It must start and end with alphanumeric characters.
 
+[[put-data-frame-transform-query-parms]]
+==== {api-query-parms-title}
+
+`defer_validations`::
+ (Optional, boolean) When `true`, this will cause deferrable validations to not run.
```

Comment:
We should include more detail about deferred validations before the second sentence. 

```suggestion
 (Optional, boolean) When `true`, this will cause deferrable validations to not run. Deferrable validations include checks for the existence of the source index and that the user has sufficient privileges, skipping these validations at creation is useful for situations where the source index does not exist but will be created later. Deferred validations are always run when the {dataframe-transform} is started.
```

Reply:
Maybe the very last sentence of that suggested change should be changed to: "Deferred validations are always run when the {dataframe-transform} is started, with the exception of privilege checks. If the user who created the transform does not have the required privileges on the source and destination indices then the transform will start but then fail when it attempts the unauthorized operation."

Reply:
These details are great, thanks!